### PR TITLE
ADX 733 New Country Estimates Dataset

### PR DESCRIPTION
**Pull request by jonathansberry**
We need to create a new version of the country estimates datasets that will have a different name and be able to store any new changes to the data schemas  for the 2022 round of estimates.

Other systems interoperable with the ADR are using the field "type" which remains unchanged for 2021 data, but is different for 2022 data.  I have asked the Naomi dev team if they intend to be interoperable with both years data or just 2022 data.  Based on their response I will make a recommendation to update their search API URL.  But what is proposed below is essentially backwards compatible from Naomi's perspective, and shouldn't change any of the existing behaviour.

### CODE REVIEW

File: package_schemas/country_estimates/2_country_estimates_22.json [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    "about": "",
+    "about_url": "http://github.com/ckan/ckanext-scheming",
+    "add_data_on_create": false,
+    "dataset_fields": [{
+            "field_name": "title",
+            "label": "Title (autogenerated):",
+            "form_snippet": "text.html",
+            "validators": "autogenerate",
+            "classes": ["hidden"],
+            "template": "{} Inputs UNAIDS Estimates 2022",
```
Comment (tomeksabala) [inline package_schemas/country_estimates/2_country_estimates_22.json]: shouldn't we get read of "UNADIS Estimates" from files & variables names all together?

File: package_schemas/country_estimates/2_country_estimates_22.json [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+            "template_formatters": ["comma_swap"],
+            "display_group": "Country Estimates 2022"
+        },
+        {
+            "field_name": "name",
+            "label": "URL",
+            "form_snippet": "text.html",
+            "validators": "autogenerate auto_create_valid_name not_empty unicode name_validator package_name_validator",
+            "classes": ["hidden"],
+            "template": "{}-inputs-unaids-estimates-2022",
```
Comment (tomeksabala) [inline package_schemas/country_estimates/2_country_estimates_22.json]: shouldn't we get read of "UNADIS Estimates" from files & variables names all together?

File: package_schemas/country_estimates/2_country_estimates_22.json [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+            "preset": "country_list",
+            "display_property": "dc:contributor",
+            "validators": "not_empty unique_combination",
+            "display_group": "Country Estimates 2022"
+        },
+        {
+            "field_name": "tag_string",
+            "label": "Tags",
+            "preset": "tag_string_autocomplete",
+            "default": "UNAIDS Inputs, ANC, ART, SVY, Country Estimates",
```
Comment (tomeksabala) [inline package_schemas/country_estimates/2_country_estimates_22.json]: shouldn't we get read of "UNADIS Estimates" from files & variables names all together?

**Review by tomeksabala — CHANGES_REQUESTED** [review]

File: package_schemas/country_estimates/2_country_estimates_22.json (hunk 1 above)
Comment (jonathansberry) [reply package_schemas/country_estimates/2_country_estimates_22.json]: Yeah excellent catch.  Thankyou.

File: package_schemas/country_estimates/2_country_estimates_22.json (hunk 2 above)
Comment (jonathansberry) [reply package_schemas/country_estimates/2_country_estimates_22.json]: Yeah excellent catch. Thankyou.

File: package_schemas/country_estimates/2_country_estimates_22.json (hunk 3 above)
Comment (jonathansberry) [reply package_schemas/country_estimates/2_country_estimates_22.json]: Less confident about this, but we don't really have any clear strategy for tagging at the moment. It probably does just make sense to drop the UNAIDS Inputs tags

**Review by tomeksabala — APPROVED** [review]

**Pull request merged**